tests/integration/test_storage.sh

```diff
@@ -35,8 +35,11 @@ setup () {
     # Perform setup routines here.
     copy /etc/sysconfig/docker-storage-setup /etc/sysconfig/docker-storage-setup.atomic-tests-backup
     TEST_DEV_1=/dev/vdb
-    findmnt -o SOURCE | grep "^$TEST_DEV_1" | uniq | xargs -r umount
-    wipefs -a "$TEST_DEV_1"
+    if findmnt ${TEST_DEV_1}; then
+        umount ${TEST_DEV_1};
+        udevadm settle
```

Comment:
is this call to udevadm needed?  I'm not seeing what it does for you.
Otherwise LGTM

Reply:
Actually, we don't need this whole chunk at all. I had left it to be compatible with whatever setup folks may have had locally, though it's rather unusual to have the block device itself be mounted. I nuked it completely below. ⬇️